### PR TITLE
Implement a feature that user can upload their photos to tencent cloud and server

### PR DESCRIPTION
**Pull request by kaigregliu**

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ services:
       - POSTGRES_PORT=5435
       - USER_TOKEN_SECRET=dummySecret
 
+
```
Comment (hcsum) [inline docker-compose.dev.yml]: plz remove extra line

File: src/migrations/1674664190143-sync.ts [hunk 2]
```diff
@@ -1,40 +0,0 @@
-import { MigrationInterface, QueryRunner } from "typeorm";
-
-export class sync1674664190143 implements MigrationInterface {
```
Comment (hcsum) [inline src/migrations/1674664190143-sync.ts]: 2023-03-01 at 16 07
we should have these migration files. why they are deleted?

File: src/migrations/1677479564056-sync.ts [hunk 3]
```diff
@@ -1,21 +1,25 @@
 import { MigrationInterface, QueryRunner } from "typeorm";
 
-export class createTableMatchingEvent1674651604338 implements MigrationInterface {
```
Comment (hcsum) [inline src/migrations/1677479564056-sync.ts]: why the new migrations are creating user table again?

**Review by hcsum — CHANGES_REQUESTED** [review]

File: docker-compose.dev.yml (hunk 1 above)
Comment (kaigregliu) [reply docker-compose.dev.yml]: Done

File: src/migrations/1674664190143-sync.ts (hunk 2 above)
Comment (kaigregliu) [reply src/migrations/1674664190143-sync.ts]: Done

File: src/migrations/1677479564056-sync.ts (hunk 3 above)
Comment (kaigregliu) [reply src/migrations/1677479564056-sync.ts]: Done

**Review by hcsum — APPROVED** [review]

**Pull request merged**